打开调试，编辑 `\config\app.php` 文件

```php
//调试，false 关闭，true 开启
'debug' => true,
```

部署到线上时，请设置为 false ，将屏蔽一切错误

**测试，制造一个异常**

```php
<?php
namespace application\home\controller;
use my\library\SmartyController;

class Index extends SmartyController{
    public function test(){
        throw new \Exception('抛出异常');
    }
}
```

浏览器打开

```php
File: /usr/share/nginx/html/mysmarty/application/home/controller/Index.php
Line: 7
Code: 0
Message: 抛出异常
Trace: #0 [internal function]: application\home\controller\Index->test() #1 /usr/share/nginx/html/mysmarty/my/includes/Start.class.php(175): call_user_func_array(Array, Array) #2 /usr/share/nginx/html/mysmarty/my/includes/Start.class.php(107): my\includes\Start::go('home', 'Index', 'test', Array) #3 /usr/share/nginx/html/mysmarty/public/index.php(8): my\includes\Start::forward() #4 {main}
```

